pkg/cmd/extension/command_test.go

```diff
@@ -1024,6 +1030,86 @@ func Test_checkValidExtension(t *testing.T) {
 	}
 }
 
+func Test_checkValidExtensionWithLocalExtension(t *testing.T) {
```

Comment:
@andyfeller How do you feel about having a separate test still, even though it's the same function now?

Reply:
I'm down for it especially as it can highlight specific problems rather than keeping it awash in a huge test table.